packages/host/app/components/matrix/room.gts

```diff
@@ -357,6 +387,15 @@ export default class Room extends Component<RoomArgs> {
     this.doSendMessage.perform(this.messageToSend, this.cardtoSend);
   }
 
+  @action
+  private sendOpenCards() {
+    for (let stackItem of this.operatorModeStateService.topMostStackItems()) {
+      if (stackItem.type == 'card') {
+        this.doSendMessage.perform('', stackItem.card);
```

Comment:
please use `undefined` instead of `''`, since `undefined` is specifically a type that is permitted as the first param.